options.go

```diff
@@ -75,3 +75,10 @@ func newApplyFuncContainer(apply func(*liteconfig.Config)) *applyFuncContainer {
 		applyInternal: apply,
 	}
 }
+
+// WithFrontendIP starts Temporal frontend service on specified address.
+func WithFrontendIP(address string) ServerOption {
```

Comment:
That doesn't seem consistent with the commenting in the rest of the file.

Reply:
Updated the suggestion, does that seem more in line with existing comments? I just wanted to provide an example like 0.0.0.0 and state what the behavior is when the option is not specified.

Reply:
Updated, though it didn't seem like this would be a library function that would be expected to be called by a user.
These constraints do get checked in the `urfave` Before callback.